lib/ansible/modules/network/sda/sda_rest.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: sda_rest
+
+short_description: Direct access to the Cisco Software-Defined Access (SDA) controller
+
+version_added: "2.6"
+
+description:
+    - Enables management of the Cisco Software-Defined Access controller, more commonly called DNA Center
+    - More information about the DNA Center API can be found at U(https://developer.cisco.com/site/dna-center-rest-api/)
+
+options:
+    url_username:
+        description:
+            - The username for use in HTTP basic authentication.
+        required: true
+    url_password:

```

Comment:
I would go for **password** here, it's what most modules do.

Reply:
I chose url_password since that's what the `uri` documentation shows, but changed to password since I think it makes more sense and it's more consistent.